analysis/SimulationDataset.py

```diff
@@ -92,11 +91,11 @@ def SampleOneEvent( self, EnergyResolution=0.0, TimeResolution=0.0 ):
 
         modifiedEvent = []
         for photon in self.inputData[ eventID ]:
-
-          newEnergy = photon[1] * ( 1 + np.random.normal( 0.0, EnergyResolution ) ) # Energy resolution as a percentage
-          newTime = photon[2] + ( np.random.normal( 0.0, TimeResolution ) ) # Time resolution as absolute ns
-          modifiedEvent += [[ photon[0], newEnergy, newTime, photon[3], photon[4], photon[5] ]]
-
+          # print("photon coords = ", photon)
```

Comment:
Avoid leaving debug code - commented-out or not

Reply:
I overlooked it. Both comments are removed now.